src/index.js

```diff
@@ -0,0 +1,102 @@
+import './style.css';
+// import { format } from 'date-fns';
+import * as dom from './dom';
+import createTodoList from './createTodoList';
+import './addTodo';
+import deleteButton from './deleteButton';
+import clearButton from './clearButton';
+import renderProjects from './renderProjects';
+import todoItem from './todoItem';
+
+// CREATE FIRST PROJECTS
+
+let projects = [
+  {
+    name: 'Your first project',
+    todoList: [
+      {
+        itemTitle: 'Your first todo',
+        itemDescription: 'Finish this',
+        itemDueDate: new Date(2021, 8, 19),
+        itemPriority: 1,
+        itemStatus: false,
+      },
+    ],
+  },
+];
+// GET FROM STORAGE
+const getProjects = () => {
+  const storage = JSON.parse(localStorage.getItem('projects'));
+  if (storage) {
+    projects = [];
+    projects = projects.concat(storage);
+  }
+};
+// ADD PROJECTS TO SUBMIT DROPDOWN
+const addProjectsToDropdown = () => {
+  if (projects) {
+    const select = document.getElementById('projects');
+    select.innerHTML = '';
+    projects.forEach((project) => {
+      select.add(new Option(project.name));
+    });
+  }
+};
+// FIND BUTTONS
+const findProjectButtons = () => {
+  const projectButtons = document.querySelectorAll('.project-btn');
+  projectButtons.forEach((projectButton, index) => {
+    projectButton.addEventListener('click', () => {
+      const todoItemsList = dom.myQuery('.todo-items-list');
+      todoItemsList.innerHTML = '';
+      projects[index].todoList.forEach((todoList) => {
+        todoItemsList.append(createTodoList(projects, todoList));
+      });
+      const buttonsDiv = dom.myQuery('.buttons');
+      buttonsDiv.innerHTML = '';
+      clearButton(projects, projectButton);
+      deleteButton(projects, projectButton);
+    });
+  });
+};
+// ADD PROJECT
+const addProject = dom.myQuery('.new-project-btn');
+addProject.addEventListener('click', (e) => {
+  e.preventDefault();
+  const projectName = dom.myQuery('#new-project').value;
+  if (projectName) {
+    const project = {
+      name: projectName,
+      todoList: [],
+      id: new Date(),
+    };
+    projects.push(project);
+  }
+  addProjectsToDropdown();
+  renderProjects(projects);
+  findProjectButtons();
+  localStorage.setItem('projects', JSON.stringify(projects));
+});
+
+const submitButton = () => {
```

Comment:
Let's improve this codebase, right after a user clicked submit for either for creation of the project or todo, the input field should be erased or cleared.

Example:
I want to create a new project;
In the input text, I typed Frontend, if I want to create a new project, I would have to erase the previous Frontend but I could input new text.

Reply:
Clear the input field after the creation of a project or todo. Let us address this comment